tests/stub/tx_run/test_tx_run.py

```diff
@@ -343,7 +341,7 @@ def test_should_prevent_run_after_tx_termination_on_run(self):
         self._assert_is_client_exception(exc)
 
         with self.assertRaises(types.DriverError) as exc:
-            tx.run("invalid")
+            tx.run("RETURN 1 AS n")
```

Comment:
Note this change.

I felt it was pointless to send a query here which we know will fail with exactly that Syntax error. The driver should throw even before attempting to send anything.